release_management/create_release.py

```diff
@@ -317,10 +319,12 @@ def tag_python_library(namespace, name):
         if namespace.no_release:
             return
 
-        dry_run_caller(
-            "__main__._push_tag", lambda: _push_tag(repository.geturl(), tag)
+        push_git_url = (
+            repository.geturl() if namespace.git_url is None else namespace.git_url
```

Comment:
^git_url^git_repo to make parser specification.

Reply:
Resolved.